README.md

```diff
@@ -150,3 +150,19 @@ Thanks also go to the developers of the [msvc-dev-cmd](https://github.com/ilammy
     <img valign="middle" height="100" src=".graphics/sbml-logo-only.png">
   </a>
 </div>
+
+Research notice
+~~~~~~~~~~~~~~~
+
+Please note that this repository is participating in a study into
+sustainability of open source projects. Data will be gathered about this
+repository for approximately the next 12 months, starting from June
+2021.
+
+Data collected will include number of contributors, number of PRs, time
+taken to close/merge these PRs, and issues closed.
+
+For more information, please visit `the informational
+page <https://sustainable-open-science-and-software.github.io/>`__ or
```

Comment:
again, since we are using markdown the RST syntax wont work here to resolve the link